nin/dasBoot/LayerManager.js

```diff
@@ -124,6 +136,22 @@ LayerManager.prototype.updateActiveLayersList = function(frame, forceUpdate) {
   }
 };
 
+LayerManager.prototype.showCameraPathVisualizations = function(shouldShow) {
+  for(var i = 0; i < this.layers.length; i++) {
```

Comment:
I can't see in what build step the linter is used?
It doesn't complain during project compilation.